collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryViewModel.java

```diff
@@ -168,7 +165,7 @@ public void logFormEvent(String event) {
     }
 
     public boolean hasBackgroundRecording() {
-        return preferencesProvider.getGeneralSharedPreferences().getBoolean("background_audio_recording", false);
+        return formController.getFormDef().hasAction("recordaudio");
```

Comment:
Can we use the constant from `RecordAudioActionHandler`?

Reply:
Yup! Missed that.